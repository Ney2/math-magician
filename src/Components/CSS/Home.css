@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');

body {
  margin: 0;
  padding: 0;
  background-image: url('../image/bg.jpg');
  background-repeat: no-repeat;
  background-size: cover;
}

.home {
  font-family: 'Roboto', sans-serif;
  margin-left: 24px;
  text-align: center;
  margin-top: 10%;
  color: green;
}

.intro {
  font-weight: bolder;
  font-size: 50px;
  padding: 24px;
}

.home-title {
  padding: 12px;
}

.description {
  font-size: 20px;
}

.link {
  text-decoration: none;
  margin-left: 5px;
}
