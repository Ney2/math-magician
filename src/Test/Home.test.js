import React from 'react';
import Renderer from 'react-test-renderer';
import Home from '../Components/Home';

it('renders correctly', () => {
    const tree = Renderer
        .create(<Home/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});