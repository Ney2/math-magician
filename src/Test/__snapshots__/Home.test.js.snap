// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="home"
>
  <h1
    className="intro"
  >
    Welcome to our Page!
  </h1>
  <div>
    <h3
      className="home-title"
    >
      Having a hard time with numbers?
    </h3>
    <p
      className="description"
    >
      Well! here is a solution checkout this free math magician website.
      <br />
      It is a simple website built using one of the popular JavaScript library; React.
      <br />
      This website conatins a simple calulator that performs some basic
       
      <br />
      arthematic functions like addition, subtraction, multiplication and division.
    </p>
  </div>
  <div>
    <h3
      className="home-title"
    >
      Wanna have some fun?
    </h3>
    <p
      className="description"
    >
      Just click on the Calculator link above to solve your problem.
    </p>
  </div>
  <div>
    <h3
      className="home-title"
    >
      Enjoyed the website?
    </h3>
    <p
      className="description"
    >
      You can show your support to the project by giving star on
      <a
        className="link"
        href="https://github.com/Ney2/math-magician"
      >
        GitHub.
      </a>
    </p>
  </div>
</div>
`;
